### PR TITLE
add join us page and remove submenu

### PR DESCRIPTION
**Pull request by KellyAH**
- for https://github.com/zinc-collective/www.zinc.coop/issues/103 as Zee requested in https://github.com/zinc-collective/www.zinc.coop/issues/103#issuecomment-1586255254

join us changes

### CODE REVIEW

File: src/join-us/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Join Us
+---
+
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="h-6 w-6 float-left mr-2 mt-2">
+  <path stroke-linecap="round" stroke-linejoin="round" d="M7.5 8.25h9m-9 3H12m-9.75 1.51c0 1.6 1.123 2.994 2.707 3.227 1.129.166 2.27.293 3.423.379.35.026.67.21.865.501L12 21l2.755-4.133a1.14 1.14 0 01.865-.501 48.172 48.172 0 003.423-.379c1.584-.233 2.707-1.626 2.707-3.228V6.741c0-1.602-1.123-2.995-2.707-3.228A48.394 48.394 0 0012 3c-2.392 0-4.744.175-7.043.513C3.373 3.746 2.25 5.14 2.25 6.741v6.018z" />
+</svg>
+
+Become a Co-Conspirator in our <a href="https://discord.gg/QK9VVg4D">Group Chat</a>.
```
Comment (KellyAH) [inline src/join-us/index.md:10]: Are we sure the discord invite link should be exposed here? I thought it was only accessible via invite only from existing ZC member?

File: src/join-us/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="h-6 w-6 float-left mr-2">
+  <path stroke-linecap="round" stroke-linejoin="round" d="M12 6v12m-3-2.818l.879.659c1.171.879 3.07.879 4.242 0 1.172-.879 1.172-2.303 0-3.182C13.536 12.219 12.768 12 12 12c-.725 0-1.45-.22-2.003-.659-1.106-.879-1.106-2.303 0-3.182s2.9-.879 4.006 0l.415.33M21 12a9 9 0 11-18 0 9 9 0 0118 0z" />
+</svg>
+
+Become a Sponsor in our <a href="https://opencollective.com/zinc-community#category-CONTRIBUTE">Open Collective</a>.
```
Comment (KellyAH) [inline src/join-us/index.md:16]: Is this the correct sponsor link?

File: src/join-us/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="h-6 w-6 float-left mr-2">
+  <path stroke-linecap="round" stroke-linejoin="round" d="M14.25 9.75L16.5 12l-2.25 2.25m-4.5 0L7.5 12l2.25-2.25M6 20.25h12A2.25 2.25 0 0020.25 18V6A2.25 2.25 0 0018 3.75H6A2.25 2.25 0 003.75 6v12A2.25 2.25 0 006 20.25z" />
+</svg>
+
+Become a Contributor on <a href="https://github.com/zinc-collective/convene#contributing-to-convene">Convene</a> or <a href="https://github.com/zinc-collective/www.momentpark.com#wwwmomentparkcom">Moment Park</a>.
```
Comment (KellyAH) [inline src/join-us/index.md]: [momentpark repo](https://github.com/zinc-collective/www.momentpark.com) has been archived. Do we still want to link to it here? Or do I have the wrong repo linked?

File: src/join-us/index.md (hunk 2 above)
Comment (zspencer) [reply src/join-us/index.md:16]: Yep!

File: src/join-us/index.md (hunk 3 above)
Comment (zspencer) [reply src/join-us/index.md]: I think you want to link to https://github.com/zinc-collective/mp-shake-it-photo

File: src/join-us/index.md (hunk 1 above)
Comment (zspencer) [reply src/join-us/index.md:10]: I think it's fine to do so for now, but if we get an influx of spammers we will want to rethink.

**Review by zspencer — APPROVED** [review]
Tjank you!

**Review by anaulin — APPROVED** [review]

File: src/join-us/index.md (hunk 3 above)
Comment (KellyAH) [reply src/join-us/index.md]: fixed https://github.com/zinc-collective/www.zinc.coop/pull/111/commits/d6521d178324a86d69ec2dd51aad176bf8cfee17

**Comment by KellyAH** [comment]
🚀 deployed and ✅ verified on prod 💃

**Pull request merged**